src/Database/DBA.php

```diff
@@ -49,6 +50,7 @@ public static function connect($serveraddr, $user, $pass, $db, $charset = null)
 		self::$db_name = $db;
 		self::$db_charset = $charset;
 
+		$port = 3306;
```

Comment:
I don’t think we should set the default port here in the database class. Especially if we angle towards Postgres support.

But you’re right, we shouldn’t use `$port` if it isn’t set. I suggest adding a condition closer to the mysqli invocation.

Reply:
Maybe `$port = false;` and `if ($port > 0)` ?

Reply:
Good idea.